docs/lib/debugging/dev-menu.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Developer Menu
+description: Amplify developer menu setup, activation, and usage information
```

Comment:
This description needs to be clearer about what the customer goal is. Something in the lines of... "Amplify developer menu helps you .... Learn how to setup, active and browse usage information".